client/web/src/nav/IconRadioButtons.scss

```
@@ -9,7 +9,18 @@
         align-items: center;
         height: 2.25rem;
         width: 2.25rem;
+        margin-bottom: 0;
+        border-radius: 50%;
+        box-shadow: inset 0 0 0 3px rgba(#329af0, 0);
+
+        transition: box-shadow 50ms ease-in;
+
+        &:focus-within {
+            box-shadow: inset 0 0 0 3px rgba(#329af0, .5);
```

```suggestion
            box-shadow: inset 0 0 0 3px rgba($primary, .5);
```

Based on @limitedmage's feedback, it's a SASS rgba function that doesn't support the CSS variable. I don't want to introduce a new colour variable for this specifically, so I've used `$primary` here and it seems to do the job for now. I think this can be revisited when we handle focus states across the app. 

@AlicjaSuska FYI.